src/domain/RootViewModel.js

```diff
@@ -36,6 +37,7 @@ export class RootViewModel extends ViewModel {
         this.track(this.navigation.observe("login").subscribe(() => this._applyNavigation()));
         this.track(this.navigation.observe("session").subscribe(() => this._applyNavigation()));
         this._applyNavigation(this.urlCreator.getLastUrl());
+        setTheme(await this.platform.settingsStorage.getString("theme"));
```

Comment:
Can we do this as the first thing in this method? to avoid race conditions with already showing the UI after applyNavigation.

Reply:
Made the change.